### PR TITLE
bootstrap: add missing 'return False'  in is_running()

### PR DESCRIPTION
**Pull request by Williangalvani**
not tested yet, but a docker is up at  williangalvani/blueos-bootstrap:addreturn

### CODE REVIEW

**Review by patrickelectric — APPROVED** [review]

File: bootstrap/test_bootstrap.py [hunk 1]
```diff
@@ -160,23 +171,30 @@ def test_start_core_no_config(self) -> None:
...
+    def test_is_running(self) -> None:
         fake_client = FakeClient()
         bootstrapper = Bootstrapper(fake_client, FakeLowLevelAPI())
+
+        fake_client.set_active_dockers([])
         assert bootstrapper.is_running("core") is False
+
         fake_core = FakeContainer(Bootstrapper.CORE_CONTAINER_NAME)
         fake_client.set_active_dockers([fake_core])
-        assert bootstrapper.is_running("core")
```
Comment (Williangalvani) [inline bootstrap/test_bootstrap.py:171]: this line should not have been removed

Comment (Williangalvani) [reply bootstrap/test_bootstrap.py:171]: nevermind, I read the patch wrong =|

**Review by patrickelectric — CHANGES_REQUESTED** [review]
It's restarting core every 12s
image

**Review by patrickelectric — APPROVED** [review]
After further testing I got it working, it was a problem with my binds

**Comment by Williangalvani** [comment]
I am actually not so sure. I'll add a test for the timeout in another pr just to be sure

**Pull request merged**